sql/driver/conn.go

```diff
@@ -55,6 +56,7 @@ func (c *conn) Prepare(query string) (driver.Stmt, error) {
 }
 
 func (c *conn) Begin() (driver.Tx, error) {
+	c.session = []byte{}
```

Comment:
Let me send you a follow up PR with some more tests.

On Thu, Jul 9, 2015 at 10:41 PM Peter Mattis <notifications@github.com>
wrote:

> In sql/driver/conn.go
> <https://github.com/cockroachdb/cockroach/pull/1657#discussion_r34324184>:
>
> > @@ -55,6 +56,7 @@ func (c *conn) Prepare(query string) (driver.Stmt, error) {
> >  }
> >
> >  func (c *conn) Begin() (driver.Tx, error) {
> > +	c.session = []byte{}
>
> You clear the session every time a transaction is started? That doesn't
> seem right. Btw, better to clear the session using c.session = nil.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1657/files#r34324184>.
>